docs/intro.md

```diff
@@ -1 +1,32 @@
-**Telescope ALBA** documentation.
+- ALBA is a system which allows a prover to _quickly and succinctly_ convince a verifier of their knowledge regarding a large number of _valid elements_, even when presenting only a _subset_.
+- It approximates a lower bound on the prover's knowledge which permits a gap between
+    - what the prover actually knows and
+    - what the verifier is convinced they know.
+- This approach enables efficient proofs, with nearly optimal proof sizes in both _noninteractive_ and _distributed_ settings.
+- Key applications:
+    - Large-scale decentralized signatures,
+    - Universal composability in general-purpose succinct proof systems.
+- Particularly, it provides communication efficiency in decentralized or blockchain contexts where multiple provers share evidence.
+
+## Overview
+- **The protocol**
+    - The protocol addresses the problem where a prover possesses a large set of verifiable evidence that would be costly to present in its entirety.
+    - The prover's goal is to convince the verifier of their knowledge by revealing only a subset of this evidence, thus achieving efficiency in both time and communication.
+    - Given a large set $S_p$ that satisfies a predicate $R$ such that $|S_p| \geq n_p$, the prover wants to convince the verifier that the set contains at least $n_f$ elements, where $n_f$ is a threshold smaller than $n_p$.
+        - This creates an _approximate_ lower bound, as the verifier is convinced that $S_p$ meets or exceeds a threshold, though the actual size might be greater.
```

Comment:
By definition the actual size might be greater. I am not sure if "approximate" refers actually to the lower bound (as in tight/loose) or the set size (`n_p` can be smaller than `|Sp|`)

Reply:
Do you have a suggestion to clarify it?

Reply:
I think @rrtoledo's question is right on spot. The term "lower bound" in the ALBA paper is unclear.  

@tolikzinovyev - can you, please, help clarify this? I am also confused on this point just like @rrtoledo and @curiecrypt ? I've raised this question multiple times but never received a crystal clear answer. I'd appreciate a concise clarification.

Reply:
The lower bound refers to the prover proving that his set has some number of elements (> n_f). It is approximate because the prover's set actually has more elements (n_p) (in other words, the lower bound is loose). I hope this clarifies.

Reply:
@djetchev There could be several reasons why you may not want `n_p =  |S_p|`:
- You are unsure of the size of `|Sp|`: for instance, in the lottery scenario, not all elements are sent, and the actual number depends on the lottery draw since it is probabilistic
- You want to (be sure to) create a proof sooner: you may not want to wait to have collected all elements of the dataset. If you start generating the proof with `n < n_p` (and `n > n_f` of course) elements, the probability of generating a proof may still be low!